### ISSUE TITLE
Concerning the result

### ISSUE THREAD

**Issue by wusx00** [wuzhen247/HUAPA#13]
I ran the original code directly, but did not achieve the results in the paper.
As to yelp13, the accuracy is 67.12 and rmse is 65.17, after 160 epochs.
As to yelp14, the accuracy and rmse achieves similar results in paper.

I'm a little confused... Is the result on yelp13 normal? What could be the factors that lead to the differences between results of the code and results in the paper?

Thank you very much!

**Comment by wuzhen247** [comment]
Thanks for your interest in our work. I remember the model doesn't achieve the best performance within 160 epochs. The hyperparameter 'num_epochs' in code is 1000. So I advise you to try more epochs.

**Comment by wuzhen247** [comment]
> I ran the original code directly, but did not achieve the results in the paper.
> As to yelp13, the accuracy is 67.12 and rmse is 65.17, after 160 epochs.
> As to yelp14, the accuracy and rmse achieves similar results in paper.
> 
> I'm a little confused... Is the result on yelp13 normal? What could be the factors that lead to the differences between results of the code and results in the paper?
> 
> Thank you very much!

Thanks for your interest in our work. I remember the model doesn't achieve the best performance within 160 epochs. The hyperparameter 'num_epochs' in code is 1000. So I advise you to try more epochs.

**Comment by wusx00** [comment]
Well I'll run for more epochs then. Thank you very much!
